shared/fs/sortbar/sort-setting-popup.js

```diff
@@ -33,8 +33,11 @@ const stylesPopup = {
   top: 64,
 }
 const stylesIcon = {
-  marginRight: globalMargins.xtiny,
+  marginRight: globalMargins.tiny,
   color: globalColors.black_75,
+  fontSize: 13,
+  marginTop: 3,
+  display: 'inline-block',
```

Comment:
This won't work on mobile.

Reply:
What's the alternative? I needed `marginTop` to work.

Reply:
It should just work. I used it in the breadcrumb positioning.

Reply:
You can always put it in a `Box`.

Reply:
You mean like this?

```
<Box style={{marginTop: 3}}>
  <Icon ... />
</Box>
```

It doesn't work on `Icon` directly since the style is applied on the `<span>` element which is inline by default.

Reply:
Yep